### PR TITLE
[16.0][OU-ADD] payment_custom: Renamed from payment_transfer

### PR DESCRIPTION
**Pull request by carmenbianca**
Depends on #3918

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -552,7 +552,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | payment_buckaroo                                |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |new| payment_custom                            |                      |                                                 |
+| |new| payment_custom                            |                      |Renamed from payment_custom.                     |
```
Comment (legalsylvain) [inline docsource/modules150-160.rst]:
```suggestion
| |new| payment_custom                            |                      |Renamed from payment_transfer.                     |
```

I guess...

**Comment by legalsylvain** [comment]
/ocabot migration payment_transfer

<dependency>Depends on : 

- payment : #3918</dependency>

File: docsource/modules150-160.rst (hunk 1 above)
Comment (pedrobaeza) [reply docsource/modules150-160.rst]: It should include the word `Done` for getting it tested in the CI:
 
```suggestion
| |new| payment_custom                            |                      | Done. Renamed from payment_custom.               |
```

Comment (legalsylvain) [reply docsource/modules150-160.rst]: The second column (status) is parsed, not the third(extra description) 
Except that, you're right.

Comment (carmenbianca) [reply docsource/modules150-160.rst]: Done, thank you :+1:

**Comment by carmenbianca** [comment]
This is basically done, but depends on #3918, in which I've identified some problems.

**Comment by legalsylvain** [comment]
could you fix pre-commit ?

File: docsource/modules150-160.rst [hunk 2]
```diff
@@ -576,7 +576,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | payment_stripe                                  |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| payment_transfer                          |                      |                                                 |
+| |del| payment_transfer                          |                      |Renamed to payment_custom.                       |
```
Comment (carmenbianca) [inline docsource/modules150-160.rst]: I should also label this DOne.

**Comment by carmenbianca** [comment]
> could you fix pre-commit ?

@legalsylvain Done, thank you.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by kevinberger-ch** [comment]
Hi,

I am an absolute beginner with OpenUpgrade. I noticed that the `post-migration.py` script from this module threw the following error: `ERROR: column "provider" does not exist`. When I replaced `WHERE provider = 'transfer'` with `WHERE custom_mode = 'transfer'` in the script, the upgrade worked.

Maybe someone can make use of this information.

Best regards

**Comment by pedrobaeza** [comment]
I don't think so. Your migration doesn't seem correct, as at that stage, the columns should be renamed already.

**Pull request merged**